uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Contact.php

```diff
@@ -21,4 +21,14 @@ public static function fabricate($params = []) {
 
     return array_shift($result['values']);
   }
+
+  public static function fabricateWithEmail($params, $email) {
+    $contact = self::fabricate($params);
```

Comment:
You right. That actually worked.
My initial thought was that since emails are stored in separate tables, a separate API call is needed for that.
Thanks.